src/pages/nominate.js

```diff
@@ -52,16 +54,14 @@ class App extends React.Component {
 
   handleSubmit = e => {
     const validation = this.validateName();
+    if (!validation && window.confirm('This form will take you to Twitter https://twitter.com/ in order to submit your nomination')) {
+      const urlEncodedTwitterHandle = encode(this.state.name);
```

Comment:
looks like this might be broken at the moment, so we might want to revert to the earlier version that didn't attempt to handle UTF-8.  (Language encoding is so difficult; thanks ASCII 😫)